app/src/lib/git/core.ts

```diff
@@ -101,6 +101,10 @@ export async function git(args: string[], path: string, name: string, options?:
   const defaultOptions: IGitExecutionOptions = {
     successExitCodes: new Set([ 0 ]),
     expectedErrors: new Set(),
+    env: {
+      GIT_TERMINAL_PROMPT: '0',
+      PATH: process.env.PATH || '',
```

Comment:
Fair warning, Windows might set `Path` here instead of `PATH`. I gotta test this on both platforms but [this is a hilarious/sad warning I've encountered before](https://github.com/desktop/dugite/blob/master/lib/git-environment.ts#L65-L74)